draft-ietf-rats-reference-interaction-models.md

```diff
@@ -270,9 +270,9 @@ Claim Selection ('claimSelection'):
 
 : *optional*
 
-: A (sub-)set of Claims which can be created by an Attester.
+: A set of (sub-)sets of Claims which can be created by the Attesting Environments of an Attester.
 
-: Claim Selections act as optional filters to specify the exact set of Claims to be included in Evidence. For example, a Verifier could send a Claim Selection, among other elements, to an Attester. An Attester MAY decide whether or not to provide all requested Claims from a Claim Selection to the Verifier. If there is no way to convey a Claim Selection in a remote attestation protocol, a default Claim Selection (e.g., "all") MUST be defined be the Attester and SHOULD be known to the Verifier.
```

Comment:
The following:

> _"In order to select Claims, Claims that can be potentially included in Evidence by an Attesting Environment have to be known by the Verifier."_

is either stating the obvious, in which case it can be dropped without loss of information, or it's trying to state something about the need for a discovery function that is however not fully articulated.

Reply:
Hm, yes this is kind of "water is wet" and yes it could be dropped, but we were taking great care here to avoid confusion. This is really wet water we do not have to tell you that it is wet and drop it.

Reply:
Drop it, please. Less is more :-)